### DISCUSSION TITLE
Question regarding usage of MDN descriptions for code comments

### DISCUSSION THREAD

**Discussion by jzombie** [mdn/mdn-community#285] [Content]
Hello!

I have a [package that I'm developing](https://github.com/zenOSmosis/webrtc-peer/pull/96/files) for WebRTC under MIT license.

After have reviewed https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Attrib_copyright_license I am still not clear about the following scenario and think it's best to ask.

I've borrowed a lot of comments from MDN and would like to know if this is permissible or if I need to rewrite every comment. 

This is basically a research / educational / experimental project for me and I'd like to not deviate as much as possible from the original content.

![Screen Shot 2022-11-16 at 8 21 25 AM](https://user-images.githubusercontent.com/1139657/202205510-55183608-8859-41ff-82bb-6f9c202b6900.png)

![Screen Shot 2022-11-16 at 8 26 00 AM](https://user-images.githubusercontent.com/1139657/202206551-818e7777-0824-49d5-b381-1469ef7de9e5.png)

The in-progress development PR is: https://github.com/zenOSmosis/webrtc-peer/pull/96/files

Thanks,

Jeremy

**Comment by teoli2003** [comment]
Pinging @Rumyra here.
